epam-promo/components/widgets/__tests__/Avatar.test.tsx

```diff
@@ -25,6 +26,40 @@ describe('Avatar', () => {
             />)
             .toJSON();
         expect(tree).toMatchSnapshot();
+
+    });
```

Comment:
Please move this tests to uui-components package, because this logic implements there. In skins we normally have only snapshots, but component logic we tests in that package we it implemented.